plugins/bcli.c

```diff
@@ -694,32 +594,97 @@ static struct command_result *getchaininfo(struct command *cmd,
 	return command_still_pending(cmd);
 }
 
+/* Mutual recursion. */
+static struct command_result *estimatefees_done(struct bitcoin_cli *bcli);
+
+/*
+ * Calls `estimatesmartfee` with targets 2/CONSERVATIVE (very urgent),
+ * 3/CONSERVATIVE (urgent), 4/ECONOMICAL (normal), and 100/ECONOMICAL (slow)
```

Comment:
this comment should be updated in your next commit

Reply:
I deleted it.  The code is now clear, and the comment just risks getting outdated again.